src/Nncase.Tests/Core/UnitTestBfloat16.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Canaan Inc. All rights reserved.
+// Licensed under the Apache license. See LICENSE file in the project root for full license information.
+
+using Nncase;
+using Nncase.Tests.TestFixture;
+using Xunit;
+
+namespace Nncase.Tests.CoreTest;
+
+public sealed class UnitTestBfloat16
+{
+    [Fact]
+    public void TestInfinity()
+    {
+        var positiveInfinity = (BFloat16)1F / (BFloat16)0F;
+        Assert.Equal(positiveInfinity, BFloat16.Infinity);
```

Comment:
First argument of `Assert.Equal` is expected, so this line should be `Assert.Equal(BFloat16.Infinity, positiveInfinity)`